locales/zh-CN.json

```diff
@@ -1,4 +1,5 @@
 {
+  "CMDLINE_REGISTERED": "Successfully registered application commands.",
```

Comment:
locale will already default to en-US for missing strings, there should be no changes to zh-CN